### PR TITLE
Add meditation functionality

### PR DESCRIPTION
**Pull request by khalifan-kfan**
PR Checklist (Make sure these options are checked before raising the PR)

- [ ] - Latest `master` branch merged
- [ ] - Descriptive PR title (can be used in release notes)

#### Summary of Changes (What does this PR do?)
- Adds meditation functionality and esau link

#### Is this change ready to hit production in its current state?
- [x] Yes
- [ ] No

#### Jira Ticket(s) Number
- [SA-79]
- [SA-78]

## Screenshots/Videos

image
image
image

### CODE REVIEW

File: src/pages/ResilienceActivities/Activities/Meditation/meditation.jsx [hunk 1]
```diff
@@ -0,0 +1,381 @@
+import React, { useState, useRef, useEffect, useCallback } from 'react';
+import ResilienceActivitiesPageComponent from '../..';
+import { Howl, Howler } from 'howler';
+import Modal from '../../../../components/modal/index.jsx';
+import { ReactComponent as MeditaionBigIcon } from '../../../../assets/svg/Resilience/meditationBigIcon.svg';
```
Comment (FaithDaka) [inline src/pages/ResilienceActivities/Activities/Meditation/meditation.jsx:5]: Typo here with the word Meditation

File: src/pages/ResilienceActivities/AllActivitiesPage.jsx [hunk 2]
```diff
@@ -2,18 +2,22 @@ import React from 'react';
 import ResilienceActivitiesPageComponent from '.';
 import Form from '../../components/form/Form';
 import ResilienceActivityPill from '../../components/form/ResilienceActivityPill';
+import { ReactComponent as Meditaion } from '../../assets/svg/Resilience/meditationIcon.svg';
```
Comment (FaithDaka) [inline src/pages/ResilienceActivities/AllActivitiesPage.jsx:5]: Typo here with the word Meditation

**Review by FaithDaka — CHANGES_REQUESTED** [review]
Great work @khalifan-khan 
Make changes as required. Lint your files using prettier before making commits.

File: src/pages/MedicineTracking/IntroPage.jsx [hunk 3]
```diff
@@ -12,8 +12,8 @@ const IntroPage = () => {
         <Form>
           <MedicationImg className="image" />
           <Largebtn title="Add Medicine" link="/medication/tracking" />
-          <Largebtn title="Monitor Medicines" link="/medication/assessment/0" />
-          <Largebtn title="Record Your Medicines" link="" />
+          <Largebtn title="Record Your Medication" link="/medication/assessment/0" />
```
[the remark this answers is not in the document]
Comment (khalifan-kfan) [reply src/pages/MedicineTracking/IntroPage.jsx:15]: yes it does as per the new designs

**Comment by khalifan-kfan** [comment]
Thanks for the feedback @FaithDaka, would you please re-review.

**Review by FaithDaka — APPROVED** [review]

**Pull request merged**